jenkins-slaves/ansible/roles/common/tasks/main.yml

```diff
@@ -27,6 +27,16 @@
     name: "*"
     state: latest
 
+- name: Install additional RHEL repository
+  yum:
+    name: https://packages.endpoint.com/rhel/7/os/x86_64/endpoint-repo-1.8-1.x86_64.rpm
+    state: present    
+
+- name: Install Git
+  yum:
+    pkg: ['git', 'tig']
```

Comment:
do we need `tig` for browsing node git repos? wdyt @winklerm, @AntStephenson, @almope ?

Reply:
I think we do not need it but it can be useful when you debug something on a Jenkins agent. Having said that, I have not used it that way probably yet, maybe someone else has.